docs/faq.md

```diff
@@ -8,7 +8,7 @@ starting their server. Wouldn't it be nice if this information could be
 embedded in the nbgitpuller link, so this (often confusing) choice is made
 for your students?
 
-While it would indeed be very nice, this is not currently possible for two
+While it would indeed be very nice, this is not currently not easy for two
```

Comment:
I think this should be, "this is currently not easy," to remove the double-negative.

Reply:
Thanks @ryanlovett! Fixed